schemas/blocks/list_collector.json

```diff
@@ -64,6 +64,9 @@
       "question_variants": {
         "$ref": "definitions.json#/question_variants"
       },
+      "routing_rules": {
```

Comment:
As discussed in IRL, it would be worth adding this to one of the valid test_list_collector schemas in this repo.